Cargo.toml

```
@@ -16,9 +16,14 @@ crate-type = ["staticlib", "rlib"]
 
 [dev-dependencies]
 criterion = "0.3.0"
-rand = "0.7.3"
 itertools = "0.8.2"
 
 [[bench]]
 name = "benches"
 harness = false
+
+[target.'cfg(target_arch = "wasm32")'.dev-dependencies]
+wasm-bindgen-test = "0.3.13"
```

Maybe specifying `0.3` is enough? 
```suggestion
wasm-bindgen-test = "0.3"
```

---

Changed as suggested